Example/Tests/Tests.swift

```diff
@@ -89,5 +89,115 @@ class Tests: XCTestCase {
         wait(for: [expectation], timeout: 1.0)
         connectivity.stopNotifier()
     }
-    
+
+    func testContainsStringValidation() {
+        checkValidation(
+            string: "a test",
+            matchedBy: "test",
+            expectedResult: true,
+            using: .containsExpectedResponseString
+        )
+        checkValidation(
+            string: "est",
+            matchedBy: "test",
+            expectedResult: false,
+            using: .containsExpectedResponseString
+        )
+    }
+
+    func testEqualsStringValidation() {
+        checkValidation(
+            string: "test",
+            matchedBy: "test",
+            expectedResult: true,
+            using: .equalsExpectedResponseString
+        )
+        checkValidation(
+            string: "est",
+            matchedBy: "test",
+            expectedResult: false,
+            using: .equalsExpectedResponseString
+        )
+    }
+
+    func testRegexStringValidation() {
+        checkValidation(
+            string: "test1234",
+            matchedBy: "test[0-9]+",
+            expectedResult: true,
+            using: .matchesRegularExpression
+        )
+        checkValidation(
+            string: "testa1234",
+            matchedBy: "test[0-9]+",
+            expectedResult: false,
+            using: .matchesRegularExpression
+        )
+    }
+
+    func testCustomValidation() {
+        //swiftlint:disable:next nesting
```

Comment:
I don't think this warning was aimed at hiding nested classes in tests, but I'm happy to remove this as well and just make the class private to the file, if you'd like :)